scripts/build-wasm.sh

```diff
@@ -2,7 +2,9 @@
 
 set -euo pipefail
 
-npx wasm-pack build --out-dir target/wasm-pack/build --no-typescript --target nodejs --features js
+# [SC2086] Intenitonally omitting double quotes to allow word splitting of WASM_PACK_BUILD_FLAGS
+# shellcheck disable=SC2086
+npx wasm-pack build ${WASM_PACK_BUILD_FLAGS:-} --out-dir target/wasm-pack/build --no-typescript --target nodejs --no-default-features --features js
```

Comment:
Random thought I just had: since this env var is capable of injecting arbitrary arguments into the `wasm-pack` call, is there any chance of someone being able to potentially trigger Shenanigans if they're able to influence the environment in which `build-wasm.sh` (or `test-wasm.sh`) is invoked?  Not knowing anything about `wasm-pack`, it's possible that there's no way that a command line option to `wasm-pack` could possibly cause problems, so I'll leave the call up to you, it just struck me as a potentially risky way to inject args, compared to (say) passing it on the command line, or detecting the presence of an env var (eg `WASM_PACK_NO_OPT`) and setting just the option that's needed.